index.html

```diff
@@ -846,6 +847,64 @@ <h3 class="section-subheading text-muted">Resonant's rich data visualization cap
         </div>
     </div>
 
+    <!-- Getting Started -->
+    <div class="portfolio-modal modal fade" id="getting-started" tabindex="-1" role="dialog" aria-hidden="true">
+        <div class="modal-content">
+            <div class="close-modal" data-dismiss="modal">
+                <div class="lr">
+                    <div class="rl">
+                    </div>
+                </div>
+            </div>
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-8 col-lg-offset-2">
+                        <div class="modal-body">
+                            <!-- Project Details Go Here -->
+                            <h2>Getting Started</h2>
+                            <p class="portfolio-description">
+                                Depending on your development focus, you may want to start at various entry points in
+                                Resonant. As you need additional functionality, you can branch out and integrate with more
+                                of the Resonant components.
+                            </p>
+                            <h3>Data Management</h3>
+                            <p class="portfolio-description">
+                                Our full-featured data management platform, Girder, is a great place to start. Follow the
```

Comment:
> full-featured

This phrase seems vague to me -- could we come up with a more descriptive adjective?